### PR TITLE
fix error message when querying non existent field

### LINKED ISSUE #1816: Improve display of named type for errors
If we query a non existent field, like this:

```graphql
{
  computer(id: "Computer1") {
    nonExistentField
  }
}
```

Here is the error:
```json
{
  "errors": [
    {
      "message": "invalid type error, expected another type than 'Named type Computer'"
    }
  ]
}
``` 

We should get rid of `Named type` and just display `Computer`

### PR DESCRIPTION
**Pull request by bnjjj**
close #1816

### CODE REVIEW

**Comment by abernix** [comment]
I got the changelog on this.

**Review by garypen — APPROVED** [review]

File: NEXT_CHANGELOG.md [hunk 1]
```diff
@@ -104,6 +104,23 @@ By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/p
...
+-       "message": "invalid type error, expected another type than 'Named type Computer'"
++       "message": "cannot query field 'xxx' on type 'Computer'"
+    }
+  ]
+}
+```
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/1817
+
 ### Update apollo-router-scaffold to use the published router crate [PR #1782](https://github.com/apollographql/router/pull/1782)
```
Comment (abernix) [inline NEXT_CHANGELOG.md:124]:
```suggestion
By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/1817

### Update `apollo-router-scaffold` to use the published `apollo-router` crate [PR #1782](https://github.com/apollographql/router/pull/1782)
```

File: NEXT_CHANGELOG.md [hunk 2]
```diff
@@ -104,7 +104,24 @@ By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/p
...
+### Improve error message when querying non existent field [Issue #1816](https://github.com/apollographql/router/issues/1816)
+
+When querying a non existent field you will get a better error.
+
+```patch
+{
+  "errors": [
+    {
+-       "message": "invalid type error, expected another type than 'Named type Computer'"
++       "message": "cannot query field 'xxx' on type 'Computer'"
```
Comment (abernix) [inline NEXT_CHANGELOG.md:116]: I think this more accurately reflects the new message?

```suggestion
+       "message": "Cannot query field \"xxx\" on type \"Computer\""
```

**Review by abernix — COMMENTED** [review]
Leaving the code up to others!

**Review by Geal — APPROVED** [review]

File: NEXT_CHANGELOG.md (hunk 2 above)
Comment (garypen) [reply NEXT_CHANGELOG.md:116]: I think so. That's JSON isn't it?

**Pull request merged**